apps/root/src/pages/aggregator/swap-container/components/advanced-settings/index.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import React from 'react';
+import isEqual from 'lodash/isEqual';
+import { IconButton, Badge, colors, CogIcon } from 'ui-library';
+import { useAggregatorSettingsState } from '@state/aggregator-settings/hooks';
+import { DEFAULT_AGGREGATOR_SETTINGS } from '@constants/aggregator';
+import { useThemeMode } from '@state/config/hooks';
+
+const StyledCogContainer = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: ${({ theme: { spacing } }) => `${spacing(1)} ${spacing(2)}`};
+`;
+
+const StyledIconButton = styled(IconButton)`
```

Comment:
cant you make this with `dense` prop?

Reply:
I'm not seeing dense prop available for IconButton! I've tried with size=small but keeps a little padding